code/datums/quirks/neutral.dm

```diff
@@ -34,7 +34,8 @@
 	medical_record_text = "Patient suffers from ageusia and is incapable of tasting food or reagents."
 	mail_goodies = list(/obj/effect/spawner/random/food_or_drink/condiment) // but can you taste the salt? CAN YOU?!
 
-/datum/quirk/foreigner
```

Comment:
Rather than removing a quirk wholesale, it might be better to add a "ENABLED/DISABLED" flag so that we can remove them non-intrusively. But that's a bit out of the scope of this PR and it's been waiting for approval for two weeks, so I'll leave it up to you whether or not you want to actually implement that here.

Reply:
I'll note this down for a later implementation!